src/Compilers/CompilersPackage/source.extension.vsixmanifest

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+  <Metadata>
+    <Identity Id="CompilersPackage.Roslyn.c00e9a7d-047b-4152-8cdf-8af9c14bd087" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
+    <DisplayName>CompilersPackage</DisplayName>
+    <Description xml:space="preserve">The Roslyn compilers and MSBuild integration.</Description>
+    <ShortcutPath>..\..\..\MSBuild\15.0\Bin\Foo\</ShortcutPath>
```

Comment:
Because I'm uncreative and forgot to change this after doing some local changes on my box.  

This PR is actually dead; vsix isn't an option anymore.  I'll likely keep the branch because I'm lazy though.